### PR TITLE
English translation of readme.md

### PR DESCRIPTION
**Pull request by Fustie**
Used:
my brains and english knownledge
https://www.deepl.com/

### CODE REVIEW

**Review by Mudkip — COMMENTED** [review]
А где перевод то?

Where is the actual translation? I only see Russian in the file you made.

**Comment by Fustie** [comment]
> А где перевод то?
> 
> Where is the actual translation? I only see Russian in the file you made.

https://github.com/TRIGONIM/ggram/pull/1/files/5352ee17c0f06a7c2dcfbbf41c2047ef7bbf0a2b

File: readme_us.md [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
 
-Создавайте Telegram ботов любой сложности и **запускайте их на Garry's Mod сервере**
+[https://img.shields.io/github/languages/code-size/TRIGONIM/ggram](https://img.shields.io/github/languages/code-size/TRIGONIM/ggram)
 
-<img align="right" width="300" src="https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png">
+[https://img.shields.io/github/license/TRIGONIM/ggram](https://img.shields.io/github/license/TRIGONIM/ggram)
 
-```lua
-local bot = ggram("token")
+Create Telegram bots any difficulty and start it on your own **Garry’s Mod server**
```
Comment (Mudkip) [inline readme_us.md]: Create Telegram bots **of** any difficulty and start **them** on your own **Garry’s Mod server**

File: readme_us.md [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
+![https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png](https://user-images.githubusercontent.com/9200174/135781831-dbb545a9-b3d9-4d0a-ba58-dd42935d35f0.png)
 
-bot.command("start", function(ctx)
-	ctx.reply.text("Hello @" .. ctx.from.username)
-end)
 ```
+local bot = ggram("token")bot.enable_polling() -- enables getUpdates loopbot.command("start", function(ctx)    ctx.reply.text("Hello @" .. ctx.from.username)end)
+```
+
+Extra examples can be found in directory [/bots](/lua/ggram/bots)
```
Comment (Mudkip) [inline readme_us.md]: Extra examples can be found in the [/bots](/lua/ggram/bots) directory

File: readme_us.md [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
+## 🔥 Start of using
 
-## 🔥 Начало использования
-0. У вас уже должен быть создан бот через [@BotFather](https://t.me/BotFather)
-1. Скачайте ggram с этого репозитория и установите в `/addons/ggram`
-2. Создайте файл `/addons/ggram_bots/lua/ggram/bots/anyname/_init.lua`
-3. Заполните его содержимым демонстративного бота из папки [/bots/example](/lua/ggram/bots/example), заменив токен
+1. You should have already created a bot using [@BotFather](https://t.me/BotFather).
+2. Download ggram from this repository and install it in `/addons/ggram'.
+3. Create a file `/addons/ggram_bots/lua/ggram/bots/anyname/_init.lua`.
```
Comment (Mudkip) [inline readme_us.md]: Create **the** file `/addons/ggram_bots/lua/ggram/bots/anyname/_init.lua`

File: readme_us.md [hunk 4]
```diff
@@ -1,51 +1,52 @@
...
+4. Fill it with bot example from [/bots/example](notion://www.notion.so/lua/ggram/bots/example), replacing token
```
Comment (Mudkip) [inline readme_us.md]: Copy the code of the example bot from [/bots/example](notion://www.notion.so/lua/ggram/bots/example) into the file you just created. Do not forget to replace the token.

File: readme_us.md [hunk 5]
```diff
@@ -1,51 +1,52 @@
...
 
-## 💡 Сделано на GGRAM + скриншоты
-[Здесь (клик)](https://forum.gm-donate.ru/t/idei-telegram-botov-dlya-vashego-servera/197) опубликованы ссылки на работающие боты, а также множество идей, которые можно реализовать при помощи этого фреймворка для своего Garry's Mod сервера и не только. Возможности практически безграничны.
+## 💡 Made on GGRAM + screenshots
```
Comment (Mudkip) [inline readme_us.md]: Made **with** GGRAM + screenshots

File: readme_us.md [hunk 6]
```diff
@@ -1,51 +1,52 @@
...
 
-## ✨ Особенности
-- Работает прямо с Garry's Mod сервера
-- НЕ требует никаких сторонних .dll или WEB прослоек
-- Очень минималистичный и легко расширяемый
-- [Возможность](/lua/ggram/includes/surprise) отправки анимаций, документов, изображений
-- Создан с учетом многолетного опыта создания ботов
-- Дружит с парадигмой функционального программирования
+Here [(*click*)](https://forum.gm-donate.ru/t/idei-telegram-botov-dlya-vashego-servera/197) (not for English) was uploaded links on working bots, also many ideas, which you can implement with this framework for your Garry's Mod server and not only. The possibilities are almost endless.
```
Comment (Mudkip) [inline readme_us.md]: Ready-made bots, ideas that you could implement using this framework for your Garry's Mod server, and many more interesting things are posted [here (click)](https://forum.gm-donate.ru/t/idei-telegram-botov-dlya-vashego-servera/197). The possibilities are almost endless.

File: readme_us.md [hunk 7]
```diff
@@ -1,51 +1,52 @@
...
 
-## 📚 Документация
-- 🤔 [Что такое и как использовать update, context, middleware, reply](/info/understanding_things.md) (основные сущности)
-- 🗂 [Где создавать бота, что кидать в extensions и зачем нужна includes](/info/project_structure.md)
+## ✨ Features
 
+- Works directly with Garry’s Mod servers
```
Comment (Mudkip) [inline readme_us.md]: Works directly **on your** Garry’s Mod servers

File: readme_us.md [hunk 8]
```diff
@@ -1,51 +1,52 @@
...
+- DOESN'T require any 3rd-party .dll or WEB interfaces
```
Comment (Mudkip) [inline readme_us.md]: Does not require any additional third party .dll modules or web proxies.

File: readme_us.md [hunk 9]
```diff
@@ -1,51 +1,52 @@
...
+- Very minimalistic and easily expandable
+- [Opportunity](/lua/ggram/includes/surprise) send animations, documents and images
```
Comment (Mudkip) [inline readme_us.md]: [The possibility](/lua/ggram/includes/surprise) to send animations, documents, pictures.

File: readme_us.md [hunk 10]
```diff
@@ -1,51 +1,52 @@
...
+- Created with years of experience in creating bots
```
Comment (Mudkip) [inline readme_us.md]: Developed based on years of experience in creating bots.

File: readme_us.md [hunk 11]
```diff
@@ -1,51 +1,52 @@
...
+- Friendly with the functional programming paradigm
```
Comment (Mudkip) [inline readme_us.md]: Functional programming friendly.

File: readme_us.md [hunk 12]
```diff
@@ -1,51 +1,52 @@
...
+
+## 📚 Documentation
+
+- 🤔 What is and how to use [update, context, middleware, reply](/info/understanding_things.md) (basic entities)
```
Comment (Mudkip) [inline readme_us.md]: What [update, context, middleware, reply](/info/understanding_things.md) are and how to use them  (the fundamentals).

File: readme_us.md [hunk 13]
```diff
@@ -1,51 +1,52 @@
...
+- 🗂 [Where to create bot, what to put in extensions and for what need includes](/info/project_structure.md)
```
Comment (Mudkip) [inline readme_us.md]: [Where to create a bot, what to put in the extensions folder, and why the includes folder exists](/info/project_structure.md)

**Review by Mudkip — CHANGES_REQUESTED** [review]
Подправил немного. Внеси изменения, пожалуйста.

**Comment by AMD-NICK** [comment]
Внеси изменения, предложенные @Mudkip, исправь картинки и можно думать про мердж

**Comment by Mudkip** [comment]
Bump

**Comment by AMD-NICK** [comment]
Я сильно переписал readme в целом. Нужно делать новые правки :)

**Comment by AMD-NICK** [comment]
Перевел все, что нужно силами deepl.com

**Pull request closed without merging**